Description: - Fix upstream Makefile and add hardening.
             - Fix destination install directory.
Author: Giovani Agusuto Ferreira <giovani@riseup.net>
Last-Update: 2015-09-07Index: cvsps-2.1/Makefile
===================================================================
Index: cvsps-2.1/Makefile
===================================================================
--- cvsps-2.1.orig/Makefile
+++ cvsps-2.1/Makefile
@@ -3,7 +3,7 @@ MINOR=1
 CC?=gcc
 CFLAGS?=-g -O2 -Wall 
 CFLAGS+=-I. -DVERSION=\"$(MAJOR).$(MINOR)\"
-prefix?=/usr/local
+prefix?=/usr
 OBJS=\
 	cbtcommon/debug.o\
 	cbtcommon/hash.o\
@@ -21,7 +21,7 @@ OBJS=\
 all: cvsps
 
 cvsps: $(OBJS)
-	$(CC) -o cvsps $(OBJS) -lz
+	$(CC) $(LDFLAGS) -o cvsps $(OBJS) -lz
 
 install:
 	[ -d $(prefix)/bin ] || mkdir -p $(prefix)/bin
